### ISSUE TITLE
"CRASH" on Saving

### ISSUE THREAD

**Issue by PrajjwalDatir** [microsoft/vscode#103717]
- VSCode Version: 91899dce (snap store) latest
- OS Version: Manjaro Gnome latest version. 

Steps to Reproduce:

1. create a new file using ctrl + N
2. type name to show it as a file name for eg. type abc.txt and hit ctrl + S
3. it crashes after few seconds. I recommend stopping for 30sec

Does this issue occur when all extensions are disabled?: Yes/No
Yes

I tried using code-OSS it works perfectly only issue is with the visual studio code.

**Comment by deepak1556** [comment]
Can you launch with `code --crash-reporter-directory <path>` and provide the `.dmp` file generated once the crash happens, thanks!

**Comment by JDWilson1980** [comment]
Nss(Build.quaratine.sql)
Notify (package.sql)
Sys(REM.core.linl)
Info(core.playbook.cyber)
Info(mirror.watson.net)
Repo(force.cybercomm.global)
Repo(sniff.RedSnow.mirror)
Vol(NSS.cybercore.captive)
Vol(remote.captive.nss)
Vol(stat.captive.mod
Vol(alert.captive.host)
Vol(repo.captive.mod)
Vol(user.captive.dll)
Vol(workflow.css)
Incubator.weeb.wix/cyber

**Comment by JDWilson1980** [comment]
> cyber.dmp
nss.global.brand

**Comment by KSonny4** [comment]
I have same problem. Happens on Fedora 30 as well as one Fedora 32. Installed via snap. Here is [dmp.zip](https://github.com/microsoft/vscode/files/5047983/6489cdd8-85ac-40e4-573a97b4-59f337ab.zip).

**Comment by exoticDFT** [comment]
I'm having the same issue on Fedora 32. However, this is not the only issue I have related to crashes. My code will crash if I

- Do the above
- Close the new file, without trying to save it
- Go to the menu and click
  - Open File
  - Open Folder
  - Open Workspace
  - Save workspace as
  - About

I'm certain the list goes on at this point, but I didn't try everything. I have several dump files I can attach, but I'm attaching only the one associated with this specific issue's title (ctrl+n, add text, ctrl+s).


[new_file_save_crash.tar.gz](https://github.com/microsoft/vscode/files/5100119/new_file_save_crash.tar.gz)

**Comment by ruslan944** [comment]
I had the same issue. While running by `code --verbose` i had 
```
(code:14956): GdkPixbuf-WARNING **: 10:45:02.774: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.
(code:14956): Gtk-WARNING **: 10:45:02.776: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
(code:14956): GdkPixbuf-WARNING **: 10:45:02.777: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
to make things work again for the time being.
**
```
before crash.


Fixed for me:
```
mkdir /usr/lib/x86_64-linux-gnu
ln -s /usr/lib/gdk-pixbuf-2.0 /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0
```

**Comment by exoticDFT** [comment]
> I had the same issue. While running by `code --verbose` i had
> 
> ```
> (code:14956): GdkPixbuf-WARNING **: 10:45:02.774: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory
> 
> This likely means that your installation is broken.
> Try running the command
>   gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
> to make things work again for the time being.
> (code:14956): Gtk-WARNING **: 10:45:02.776: Could not load a pixbuf from icon theme.
> This may indicate that pixbuf loaders or the mime database could not be found.
> (code:14956): GdkPixbuf-WARNING **: 10:45:02.777: Cannot open pixbuf loader module file '/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache': No such file or directory
> 
> This likely means that your installation is broken.
> Try running the command
>   gdk-pixbuf-query-loaders > /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders.cache
> to make things work again for the time being.
> **
> ```
> 
> before crash.
> 
> Fixed for me:
> 
> ```
> mkdir /usr/lib/x86_64-linux-gnu
> ln -s /usr/lib/gdk-pixbuf-2.0 /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0
> ```

This does not seem to solve my issues at all, although I do also get the warning message here.

@deepak1556 Any update on this issue? I'm currently getting several files pilled up in my vscode window that cannot be closed without crashing the program.

**Comment by KSonny4** [comment]
I resolved this by not installing VS code using snap but normally using package from VS code page.

**Comment by PrajjwalDatir** [comment]
> I resolved this by not installing VS code using snap but normally using package from VS code page.

Yeah thanks for the solution but there's still issue with snap version :)

**Comment by exoticDFT** [comment]
> I resolved this by not installing VS code using snap but normally using package from VS code page.

This solution sadly isn't the answer for the 1.47.3 version of vscode. The non-snap version, which I have is also throwing the errors.

However, with all that said, I just downloaded the newest stable version of vscode (1.48.2) and so far the issues I was seeing are now gone. So something got resolved in the newer release. I would suspect the issues would be resolved for you @PrajjwalDatir. I'm not sure which version is in the snap store at the moment, but it does seem to be newer than 1.47.3, so I would attempt an update in snap.
